nvtabular/torch_dataloader.py

```diff
@@ -44,7 +43,7 @@ def __init__(self, tensors, batch_size=1, pin_memory=False, shuffle=False):
         self.tensors = tensors
         self.batch_size = batch_size
 
-        self.num_samples = self.tensors[0].size(0)
+        self.num_samples = self.tensors[2].size(0)
```

Comment:
why this change?

Reply:
Tensors are a tuple of (cats, conts, labels). Since this change is allowing the cats/conts to be optional (and setting to None if we don't have any) - we can't take the size of those to get the number of samples. The labels always have to be there, so I'm using that instead.